WinRT.Runtime/AgileReference.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using WinRT.Interop;
+
+namespace WinRT
+{
+    public class AgileReference : IDisposable
+    {
+        private readonly static Guid CLSID_StdGlobalInterfaceTable = Guid.Parse("00000323-0000-0000-c000-000000000046");
+        private readonly IAgileReference _agileReference;
+        private readonly IGlobalInterfaceTable _git;
+        private readonly IntPtr _cookie;
+        private bool disposed;
+
+        public unsafe AgileReference(IObjectReference instance)
+        {
+            IntPtr agileReference = default;
+            IntPtr gitPtr = default;
+            Guid iid = typeof(IUnknownVftbl).GUID;
+            try
+            {
+                Marshal.ThrowExceptionForHR(Platform.RoGetAgileReference(
+                    Platform.AgileReferenceOptions.AGILEREFERENCE_DEFAULT,

```

Comment:
not a big deal, but I'd probably just drop in 0 and a comment so enum isn't needed (like CLSCTX_INPROC_SERVER)